base/meta.jl

```diff
@@ -16,7 +16,10 @@ export quot,
        ispostfixoperator,
        replace_sourceloc!,
        show_sexpr,
-       @dump
+       @dump,
+       unblock,
+       unescape,
+       uncurly
```

Comment:
Do we want to export these functions now?
Maybe keep the status quo and let them remain as internal functions.

```suggestion
       @dump
```

Reply:
Will commit your suggestions as soon as the tests have finished 🙏